### ISSUE TITLE
Color Contrast Failures

### ISSUE THREAD

**Issue by khallwork** [techlab/jquery-smartwizard#180]
**Describe the bug**
This feature is described as being "accessible" yet it fails on almost all color contrast tests. Please create an option that passes all color contrast standards for accessibility.
